javascript/javascript2/week1/hyf-bay/main.js

```diff
@@ -0,0 +1,25 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
```

Comment:
The console logs are fine for studying, but in the future consider deleting them before pushing.